crates/wasmtime/src/runtime/module.rs

```diff
@@ -319,6 +318,25 @@ impl Module {
             .compile_module()
     }
 
+    /// Creates a new WebAssembly `Module` from the given in-memory `binary`
+    /// data and DWARF pacakage binary.
+    ///
+    /// Simlilar to [`Module::from_binary`] but with the addition of the DWARF
+    /// package binary.
+    #[cfg(any(feature = "cranelift", feature = "winch"))]
+    #[cfg_attr(docsrs, doc(cfg(any(feature = "cranelift", feature = "winch"))))]
+    pub fn from_binary_and_dwp(
+        engine: &Engine,
+        binary: &[u8],
+        dwp_binary: &[u8],
+    ) -> Result<Module> {
+        crate::CodeBuilder::new(engine)
+            .wasm(binary, None)?
+            .wat(false)?
+            .dwarf_package(dwp_binary, None)?
+            .compile_module()
+    }
```

Comment:
If you're ok with it, could this be omitted? This is a convenience for `CodeBuilder` methods, and I'd personally prefer to leave combining the method to embedders where we can. (in retrospect we can probably trim the constructors of `Module` by deferring more use cases to `CodeBuilder` as well)

Reply:
sure, deleted